include/caffe/util/frcnn_utils.hpp

```diff
@@ -22,7 +22,6 @@
 #include <opencv2/core/core.hpp>
 #include <opencv2/highgui/highgui.hpp>
 #include <opencv2/imgproc/imgproc.hpp>
-#include <opencv2/contrib/contrib.hpp>
```

Comment:
so the layer actually never used this import?

Reply:
Nope

Reply:
good catch